include/drv/wd_aead_drv.h

```diff
@@ -0,0 +1,60 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+#ifndef __WD_AEAD_DRV_H
+#define __WD_AEAD_DRV_H
+
+#include "include/wd_alg_common.h"
+#include "include/wd_aead.h"
```

Comment:
please remove "include"

Reply:
I have kept the style consistent, no need to modify here